dnanexus/src/eggd_athena.sh

```
@@ -6,9 +6,9 @@ export TZ=Europe/London
 
 set -exo pipefail
 
-# set frequency of instance usage in logs to 10 seconds
+# set frequency of instance usage in logs to 5 seconds
 kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)